base/uk.ac.stfc.isis.ibex.epics/src/uk/ac/stfc/isis/ibex/epics/writing/Writable.java

```diff
@@ -43,8 +45,18 @@
      * 
      * @param value
      *            the value to write
+     * @throws IOException
+     *              if the write failed
      */
-	void write(T value);
+	void write(T value) throws IOException;
+	
+	/**
```

Comment:
Can we add something in this comment about how it should ideally not be used unless you're sure you can write.